integration/tests/days.int.test.js

```diff
@@ -9,7 +9,7 @@ const {
 } = require("../helpers.js");
 
 
-describe("Sprints Integration Tests", () => {
+describe("Days Integration Tests", () => {
```

Comment:
maybe we could say auth : admin or something in the describe block to make it explicit